referenceframe/worldstate.go

```diff
@@ -161,3 +163,40 @@ func (ws *WorldState) ObstaclesInWorldFrame(fs FrameSystem, inputs map[string][]
 	}
 	return NewGeometriesInFrame(World, allGeometries), nil
 }
+
+// BoundingBox returns a list of limits specifying max and min x and y values.
+func (ws *WorldState) BoundingBox(ctx context.Context, limits []Limit) []Limit {
+	var xSlice []float64
+	var ySlice []float64
+
+	if len(ws.obstacleNames) == 0 {
+		return limits
+	}
+
+	for _, gf := range ws.obstacles {
+		for _, geom := range gf.geometries {
```

Comment:
Won't this not take into account how large any of the obstacles are?

Reply:
hmmm, unsure what you mean.
We are not particularly concerned with "largeness". Rather we just want to compute the bounding box, so to do that we just need the min/max of X/Y for each geometry.

Reply:
The bounding box of what, exactly?

Reply:
As written, this is looking at the centerpoints of the geometries, not their most distal vertices.